chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -482,7 +487,7 @@ public boolean enterBlock(Scene scene, Ray ray, BlockPalette palette) {
       Material prevBlock = ray.getCurrentMaterial();
 
       ray.setPrevMaterial(prevBlock, ray.getCurrentData());
-      ray.setCurrentMaterial(currentBlock, implementation.getData(node));
+      ray.setCurrentMaterial(currentBlock, 0);
```

Comment:
setCurrentMaterial should probably be changed to only take one parameter

Reply:
I overloaded it now. It is still used to copy state between rays in `Scene#worldIntersection`.